<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="speechButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Condensed</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Speech-to text</string>
   </property>
  </widget>
  <widget class="QPushButton" name="textButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>350</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Condensed</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Text-to speech</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>320</y>
     <width>101</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>text.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="title_label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>0</y>
     <width>211</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>S.A.R.A</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>120</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>speaker.png</pixmap>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
